packages/default-theme/src/logic/useDomains.js

```diff
@@ -0,0 +1,81 @@
+import { computed } from "@vue/composition-api"
+import {
+  getApplicationContext,
+  useCms,
+  useNotifications,
+} from "@shopware-pwa/composables"
+import { getSeoUrls } from "@shopware-pwa/shopware-6-client"
+import { getCmsTechnicalPath } from "@shopware-pwa/helpers"
+
+// the "last-chance route always has a name starting with `all_` - nuxt default"
+const PAGE_RESOLVER_ROUTE_PREFIX = "all_"
+
+export const useDomains = (rootContext) => {
+  const { router, routing, apiInstance } = getApplicationContext(
+    rootContext,
+    "useDomains"
+  )
+
+  const availableDomains = computed(() => routing.availableDomains || [])
+  const currentDomainId = computed(() => routing.getCurrentDomain().domainId)
+  const trimDomain = (url) => url.replace(routing.getCurrentDomain().url, "")
```

Comment:
it's not reactive this way, added comments to make it computed property, so here:
```suggestion
  const currentDomainId = computed(() => routing.getCurrentDomain.value.domainId)
  const trimDomain = (url) => url.replace(routing.getCurrentDomain.value.url, "")
```

Reply:
thanks!